docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,359 @@
+                true,
+                )
+        .unwrap()
+        .extract_v1()
+        .unwrap();
+// }
+```
+
+##### 3.6 Send the request 
+
+```rust
+// ...
+// pub fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+    // ...
+        let res = https_agent()
+        .post(req.url.as_str()) // pj_uri.endpoint
+        .header("Content-Type", "text/plain")
+        .body(String::from_utf8(req.body).unwrap())
+        .send()
+        .unwrap();
+    let res = res.text().unwrap();
+// }
+```
+
+##### 3.7 Process the response
+
+Validate PSBT inputs, outputs and fees as it may have been altered by
+the receiver.
+
+```rust
+// ...
+// pub fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+    // ...
+    let psbt = ctx.process_response(&mut res.as_bytes()).unwrap();
+
+// }
+```
+
+##### 3.8 Sign and finalize the Payjoin Proposal PSBT
+
+In the previous step we validated the PSBT and now its time finish the
+transaction, sign the PSBT and broadcast it.
+
+```rust
+// ...
+// pub fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+    // ...
+        let psbt = sender_wallet
+        .wallet_process_psbt(&base64::encode(psbt.serialize()), None, None, None)
+        .unwrap()
+        .psbt;
+    let tx = sender_wallet
+        .finalize_psbt(&psbt, Some(true))
+        .unwrap()
+        .hex
+        .unwrap();
+    let txid = sender_wallet.send_raw_transaction(&tx).unwrap();
+    txid
+// }
+```
+
+##### 3.9 And lets call the sender functions in the `main`.
+
+In the code below, `args` also read from the command line to consume the URI
+and utilise `payjoin::Uri::try_from` to convert the string to a valid URI.
+
+```rust
+fn main() {
+    let args: Vec<String> = std::env::args().collect();
+    let payjoin_uri: String = args[1].parse().unwrap();
+    let payjoin_uri = payjoin::Uri::try_from(payjoin_uri)
+        .unwrap()
+        .assume_checked();
+    let txid = sender::try_payjoin(payjoin_uri);
+    dbg!(&txid);
+}
+```
+
+## 4. Receiver
+
+To setup the receiver, we will use `payjoin-cli` a cli payjoin implementation
+developed by the `rust-payjoin` team. We will setup this receiver, generate
+BiP21 URI and go back to the `sender` we implemented and give it the URI.
+
+Open your terminal, and:
+
+#### 4.1 Install `payjoin-cli`
+
+```shell
+cargo install payjoin-cli --version 0.0.4-alpha --features danger-local-https
+```
+
+#### 4.2 Run `payjoin-cli` as the receiver
+
+```rust
+payjoin-cli -r http://localhost:18443/wallet/receiver --rpcuser foo --rpcpass qDDZdeQ5vw9XXFeVnXT4PZ--tGN2xNjjR4nrtyszZx0= receive --endpoint https://localhost:3000 1230000000
+```
+
+Once you run the the above command, an https server will start and a
+payjoin URI will be printed, and will look like this:
+
+```shell
+bitcoin:bcrt1qstujjzput7eznwv27e7qh7lpkgz33fh39g2gpt?amount=12.3&pj=https://localhost:3000/&pjos=0
+```
+
+Copy the payjoin uri as we will need it in the next(final) step.
```

Comment:
```suggestion
Copy the payjoin uri as we will need it in the next (final) step.
```

I'm familiar with a space outside parens in a written clause being convention. Obviously not in maths/science function calls.
https://www.scribens.com/grammar-rules/typography/spacing.html